src/Apps/Search/routes.tsx

```diff
@@ -7,23 +7,37 @@ import { SearchResultsCollectionsRouteFragmentContainer as SearchResultsCollecti
 import { SearchResultsGalleriesRouteRouteFragmentContainer as SearchResultsGalleriesRoute } from "Apps/Search/Routes/Galleries/SearchResultsGalleries"
 import { SearchResultsShowsRouteRouteFragmentContainer as SearchResultsShowsRoute } from "Apps/Search/Routes/Shows/SearchResultsShows"
 import { RouteConfig } from "found"
+import qs from "qs"
+import React from "react"
 import { graphql } from "react-relay"
+import { Provider } from "unstated"
+import { FilterState } from "./FilterState"
 import { SearchAppFragmentContainer as SearchApp } from "./SearchApp"
 
 // FIXME: The initial render includes `location` in props, but subsequent
 // renders (such as tabbing back to this route in your browser) will not.
 const prepareVariables = (params, props) => {
-  const paramsFromUrl = props.location ? props.location.query : {}
+  let paramsFromUrl = props.location ? props.location.query : {}
   if (
     Object.keys(paramsFromUrl).length === 0 &&
     Object.keys(params).length === 0
   ) {
-    return { term: "andy" }
+    paramsFromUrl = qs.parse(location.search.replace(/^\?/, ""))
+
+    // FIXME: This snippet only is valid during storybook development.
+    // Optionally remove this when feature is launched.
+    if (!paramsFromUrl.term && process.env.NODE_ENV === "development") {
```

Comment:
This is a bit unfortunate. In storybook development, with a nonrealistic storybook URL, we can't do the `qs.parse` stuff above. However, in Force, this all works fine (it's all a workaround for an edge case of navigating back to a tab and the incoming params being blank, that's a `FIXME` a few lines up).

So, this snippet is just for storybook development. If people are fine with developing directly in Force via `yarn link`, and/or are sketched out by including this snippet, we can always remove it once we launch. Otherwise we can just keep it with the `FIXME`.

cc @damassi maybe we can pair or take a quick look together at some point? This isn't a blocker, but I recall you noticing this as an edge case in `found` on the artist page tab navigation.

Reply:
@mzikherman - let's def pair on this as it's imperative that we work within found router. The approach currently in force surrounding the /artist/id is incorrect, and needs to be refactored out.